Core/Src/spi.c

```diff
@@ -52,15 +52,16 @@ void MX_SPI1_Init(void)
   hspi1.Instance = SPI1;
   hspi1.Init.Mode = SPI_MODE_MASTER;
   hspi1.Init.Direction = SPI_DIRECTION_2LINES;
-  hspi1.Init.DataSize = SPI_DATASIZE_8BIT;
-  hspi1.Init.CLKPolarity = SPI_POLARITY_LOW;
-  hspi1.Init.CLKPhase = SPI_PHASE_1EDGE;
+  hspi1.Init.DataSize = SPI_DATASIZE_16BIT;
```

Comment:
Should this be 8bit?
<img width="491" alt="image" src="https://github.com/lhr-solar/DataAcq-PeripheralSOM/assets/36647587/a579a0cc-bea5-4be6-8400-a22ed0eedeca">